Training/Program.cs

```diff
@@ -25,25 +23,25 @@ internal class Program {
             else if (inputStr.Length == 1)
                Console.WriteLine ($"And the winner is: {inputStr.ToUpper ()} with {inputStr.Length} vote\nPress '~' and 'enter' to Quit");
             else if (Regex.IsMatch (inputStr, "^[a-zA-Z]+$")) {
-               string winner = VotingContest (ref inputStr);
-               Console.WriteLine ($"And the winner is: {winner.FirstOrDefault ()} with {winner.Length} votes");
+               (string, int) winner = VotingContest (inputStr);

```

Comment:
Use any name for this variable declaration, it will be good to identify that instead of Item1 and Item2.
```
(char winner, int count) winner = VotingContest (inputStr);
Console.WriteLine ($"And the winner is: {winner.winner} with {winner.count} votes");
```